tests/integration/ovs_test.py

```diff
@@ -103,3 +117,44 @@ def test_create_and_remove_ovs_bridge_with_internal_port_and_static_ip():
 
     assertlib.assert_absent(BRIDGE1)
     assertlib.assert_absent(PORT1)
+
+
+@pytest.mark.xfail(raises=NmstateLibnmError, strict=True)
+def test_nm_ovs_plugin_missing():
+    with disable_nm_ovs_plugin():
+        libnmstate.apply(
+            {
+                Interface.KEY: [
+                    {
+                        Interface.NAME: BRIDGE1,
+                        Interface.TYPE: InterfaceType.OVS_BRIDGE,
+                        Interface.STATE: InterfaceState.UP,
+                    }
+                ]
+            }
+        )
+
+
+@contextmanager
+def disable_nm_ovs_plugin():
+    libcmd.exec_cmd(DNF_REMOVE_NM_OVS_CMD)
+    libcmd.exec_cmd(SYSTEMCTL_RESTART_NM_CMD)
+    libnmstate.nm.checkpoint._nmdbus_manager = None
```

Comment:
This should probably have a comment to make it easier to understand why it is happening here in the future.

Reply:
Done!

Reply:
It is inappropriate to touch a private variable, this should have been handled at each transaction or by a dedicated API of nmstate.
The fact that the tests have a problem is a hint that the user may have the same problem.

I also do not understand why this is a checkpoint only problem. Is it because the rest of the code is using libnm and the checkpoint is the only part that uses dbus directly?

I am ok with opening an issue on this and handling it in a separate PR.

Reply:
https://github.com/nmstate/nmstate/pull/654